src/components/gridHOC.jsx

```diff
@@ -93,63 +110,132 @@ export default (Grid: StaticDatagrid) => class GridHOC extends Component<Props,
   }
 
   updateGridState(updateState: UpdateStateFunctionType) {
-    const data = updateState(this.state.store);
+    let data = this.state.store.getData();
+    if (this.state.activeColumn) {
+      data = this.state.sortedData;
+    } else if (this.props.searchable) {
+      data = this.state.filteredData;
+    }
```

Comment:
This will either use sort or search, not both. Users might want to filter and then sort. 
Should instead modify this to a pipeline of data modifications. All mods should go through 
sort() => filter() => updateState();
sort() and filter can be no-op functions if not specified for usage by props.